README.md

```diff
@@ -27,6 +27,335 @@ use Utopia\Orchestration\Orchestration;
+
+    - `stdout` [String] [Reference]
+
+        The variable to store the stdout of the command in.
+
+    - `stderr` [String] [Reference]
+
+        The variable to store the stderr of the command in.
+
+    - `env` [Array]
+
+        The environment variables to set while executing the command.
+
+    - `timeout` [Integer]
+
+        The timeout in seconds to wait for the command to finish.
+
+    </details>
+
+- ### Removing a container
+
+    This method removes a container and returns a boolean value indicating if the container was removed successfully.
+
+    ```php
+    $orchestration->remove('container_id', true);
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `container_id` [String] [Required]
+
+        The ID of the container to remove.
+
+    - `force` [Boolean]
+
+        Whether to force remove the container.
+
+    </details>
+
+- ### List containers
+    
+    This method returns an array of containers.
+
+    ```php
+    $orchestration->list(['label' => 'value']);
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `filters` [Array]
+
+        Filters to apply to the list of containers.
+
+    </details>
+
+- ### List Networks
+    
+    This method returns an array of networks.
+
+    ```php
+    $orchestration->listNetworks();
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    This method has no parameters
+
+    </details>
+
+- ### Create a Network
+    
+    This method creates a new network and returns a boolean value indicating if the network was created successfully.
+
+    ```php
+    $orchestration->createNetwork('name', false);
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `name` [String] [Required]
+
+        The name of the network.
+
+    - `internal` [Boolean]
+
+        Whether to set the network to be an internal network.
+
+    </details>
+
+- ### Remove a Network
+
+    This method removes a network and returns a boolean value indicating if the network was removed successfully.
+
+    ```php
+    $orchestration->removeNetwork('network_id');
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `network_id` [String] [Required]
+
+        The ID of the network to remove.
+
+    </details>
+
+- ### Connect a container to a network
+
+    This method connects a container to a network and returns a boolean value indicating if the connection was successful.
+
+    ```php
+    $orchestration->connect('container_id', 'network_id');
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `container_id` [String] [Required]
+
+        The ID of the container to connect to the network.
+
+    - `network_id` [String] [Required]
+
+        The ID of the network to connect to.
+
+    </details>
+
+- ### Disconnect a container from a network
```

Comment:
In preview, this looks weird, but I can't see a problem... Can you?

![CleanShot 2022-03-22 at 08 59 17@2x](https://user-images.githubusercontent.com/19310830/159434257-39cadecf-e33e-446f-b421-1c940e6dbd0c.png)

Reply:
Maybe because it's under a bullet list?

Reply:
Fixed 👍